modules/standard/IO.chpl

```diff
@@ -2229,7 +2229,7 @@ inline proc channel.lock() throws {
 /*
    Release a channel's lock.
  */
-inline proc channel.unlock() {
+ proc channel.unlock() {
```

Comment:
Nit, but I think this has left an extra space in front of `proc` in a bunch of places.

Reply:
Shoot, thanks for pointing that out...  Should be fixed in https://github.com/chapel-lang/chapel/pull/16887/commits/4fb95056cb7bf26e371f9ad4ea64acf79b6ef75d (but I'll take one more pass to double-check).